include/kodlab_mjbots_sdk/joint_base.h

```diff
@@ -175,7 +180,7 @@ class JointBase {
          * 
          * @return const float& 
          */
-        const float& get_servo_torque_reference() const {return servo_torque_; }
```

Comment:
Later this is used in the robot base to populate a vector called torque_cmd. As far as I can tell that is the only place this is used. Thoughts on this change @jdcaporale @ethanmusser .

Reply:
Ethan and I were confused by my nomenclature and our output once. 
I'm happy with the name change but is the variable change correct?

Reply:
yup.